spec/std/random_spec.cr

```diff
@@ -176,4 +176,86 @@ describe "Random" do
       rng.rand(Int8::MIN..Int8::MAX).should eq expected
     end
   end
+
+  describe "random_bytes" do
+    it "gets random bytes with default number of digits" do
+      bytes = TestRNG.new(RNG_DATA_32).random_bytes
+      bytes.size.should eq(16)
+    end
+
+    it "gets random bytes with requested number of digits" do
+      bytes = TestRNG.new(RNG_DATA_32).random_bytes(50)
+      bytes.size.should eq(50)
+    end
+
+    it "fully fills a large buffer" do
+      bytes = TestRNG.new(RNG_DATA_32).random_bytes(10000)
+      bytes[9990, 10].should_not eq(Bytes.new(10))
+    end
+
+    it "fills given buffer with random bytes" do
+      bytes = Bytes.new(2000)
+      TestRNG.new(RNG_DATA_32).random_bytes(bytes)
+      bytes.size.should eq 2000
+      bytes[1990, 10].should_not eq(Bytes.new(10))
```

Comment:
These specs don't really check anything.
A lot to learn from https://github.com/crystal-lang/crystal/pull/3434/files#diff-0e9f78d10caafce0823cedec53f1c6f0